evap/static/css/_vote-colors.scss

```diff
@@ -0,0 +1,122 @@
+$vote-colors: (
+    green: $evap-green,
+    lime: $evap-lime,
+    yellow: $evap-yellow,
+    orange: $evap-orange,
+    red: $evap-red,
+    gray: $medium-gray
+);
+
+$likert: 35rem;
+
+.vote-btn {
+    height: 38px;
+    padding: 0.25rem 1rem;
+    font-size: 10pt;
+    white-space: normal;
+    line-height: 1.1;
+    border-color: $medium-gray;
+    width: $likert / 5;
+    &:last-child {
+        width: 4rem;
+        padding: 0.25rem;
+    }
+    .vote-type-bipolar &:not(:last-child) {
+        width: $likert / 7;
+        padding: 0.25rem;
+    }
+}
+.vote-btn-text {
+    position: relative;
+    top: -1px;
+}
+
+@each $name, $color in $vote-colors {
+    .vote-btn-#{$name} {
+        background: linear-gradient(to top, 5% $color, 5% transparent);
+        background-color: change-color($color, $lightness: 96%);
```

Comment:
maybe `97%`?

Reply:
Changed as the background is now kept gray